src/commands/logout.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { URL } from 'url';
+import { Flags } from '@oclif/core';
+import { AnyJson } from '@salesforce/ts-types';
+
+import SfCommand from '../sf-command';
+import { logout } from '../utils';
+
+export default class Logout extends SfCommand {
+  public static description = `logout of a Salesforce account or enviornment
+
+  Logout of https://login.salesforce.com. To logout of different providers (Salesforce org, heroku, commerce cloud, mulesoft, etc) sepecify the domain or login url of the service. For example, '--instance-url=heroku.com'. 
+  `;
+
+  public static examples = [
+    'sf logout --instance-url https://<mydomain>.my.salesforce.com',
+    'sf logout --instance-url heroku.com',
+  ];
+
+  public static flags = {
+    'client-id': Flags.string({
```

Comment:
You don't need these flags to logout. Instead of instanceurl, you could make the command just log out of everything, similar to `sf reset`. It should probably prompt too? `Are you sure...`